README.md

```diff
@@ -129,6 +129,8 @@ function initAnswers() {
     apiKey: '<API_KEY_HERE>',
     // Required, the key used for your Answers experience
     experienceKey: '<EXPERIENCE_KEY_HERE>',
+    // Optional, visitor interacting with the experience, see Visitor Configuration below
+    visitor: '<VISTOR_HERE>',
```

Comment:
since the visitor is supposed to be an object, let's just put some example visitor, or even just the example you have below, instead of `<VISITOR_HERE>. `<API_KEY_HERE> and `<EXPERIENCE_KEY_HERE>` are like that because they're strings, but the visitor is supposed to be an object.

Reply:
I decided to keep it below as well because there are additional details